test/instructions_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+class DummyClass
+  # This is the class referenced in the test data.
+end
+
+describe ScraperTest::Instructions do
+  let(:instructions) { ScraperTest::Instructions.new(file) }
+
+  describe 'valid instructions' do
+    let(:file) { './test/data/valid_instructions.yml' }
+
+    it 'returns a url' do
+      instructions.url.must_equal 'http://majles.marsad.tn/2014/fr/assemblee'
+    end
+
+    it 'returns a class to test' do
+      instructions.class_to_test.must_equal DummyClass
+    end
+
+    it 'returns the expected data' do
+      instructions.expected.must_equal(id: 34)
+    end
+  end
+
+  describe 'invalid instructions' do
+    let(:file) { './test/data/invalid_instructions.yml' }
+
+    it 'raises an error when no url is provided' do
+      assert_raises(StandardError) { instructions.url }
```

Comment:
The rest of the tests use the spec-style `foo.must…` syntax, so it's a little jarring to switch back to `assert…` style tests here. Probably best to keep this consistent:

```ruby
  -> { instructions.url }.must_raise StandardError
```